devices/silabs.js

```diff
@@ -0,0 +1,35 @@
+const exposes = require('zigbee-herdsman-converters/lib/exposes');
+const fz = {...require('zigbee-herdsman-converters/converters/fromZigbee'), legacy: require('zigbee-herdsman-converters/lib/legacy').fromZigbee};
+const tz = require('zigbee-herdsman-converters/converters/toZigbee');
+const reporting = require('zigbee-herdsman-converters/lib/reporting');
+const extend = require('zigbee-herdsman-converters/lib/extend');
+const e = exposes.presets;
+
+module.exports = [
+    {
+        zigbeeModel: ['UT-02'],
+        model: 'EFR32MG21',
+        vendor: 'Custom devices (DiY)',
```

Comment:
Please change the vendor to `Sillicon Labs`. Also make a pr for https://github.com/koenkk/zigbee2mqtt.io and explain what hardware can be used/firmware to flash.